static/app/components/events/viewHierarchy/wireframe.tsx

```diff
@@ -50,9 +49,9 @@ function Wireframe({hierarchy, selectedNode, onNodeSelect, project}: WireframePr
     () =>
       getHierarchyDimensions(
         hierarchy,
-        ['flutter', 'dart-flutter'].includes(project?.platform ?? '')
+        system === 'flutter' || system.includes('absolute')
```

Comment:
Just want to make sure that `flutter` and `dart-flutter` projects stay absolutely positioned with this change, do they both have `system` set to `flutter`? I agree with removing the project platform check though, good call!

Reply:
> do they both have `system` set to `flutter`?

Yep, it looks like `sentry-dart` uses `flutter` for the `rendering_system` too. 
https://github.com/getsentry/sentry-dart/blob/ec50b2159d85b69bbbb3619960b3f922ec5592f5/dart/test/protocol/sentry_view_hierarchy_test.dart#L25

who could we ping to triple check?

Reply:
Thanks for checking that out, I think the last step for this is to have in docs or mentioned somewhere that to use absolute rendering in the UI, the `rendering_system` must contain `absolute`, but obviously that is outside the scope of this PR. Approving!

Reply:
I will PR the original RFC because it doesn't look like this is documented elsewhere yet

Reply:
I'm working on Flutter, yes the rendering_system is always set to `flutter`: https://github.com/getsentry/sentry-dart/blob/ea6d86d569f24c0dd33f13fe51baa9b53c770d2a/flutter/lib/src/view_hierarchy/sentry_tree_walker.dart#L242

so looks good!

Reply:
PR to update the RFC is [here](https://github.com/getsentry/rfcs/pull/144)

Reply:
I'll also update the rendering system to absolute on flutter

Reply:
I think we might need to be careful about changing existing `rendering_system` because the Sentry frontend uses it to fetch icons or something 🤔

Reply:
Oh, it looks like `platform` is used for the icon:
https://github.com/getsentry/sentry/blob/f346011f0c5554e2e37509aa3c0a3d54ddbb9593/static/app/components/events/viewHierarchy/renderingSystem.tsx#L29

However, the `system` does end up in the tooltip and we might want to hide the `absolute` detail from users:
https://github.com/getsentry/sentry/blob/f346011f0c5554e2e37509aa3c0a3d54ddbb9593/static/app/components/events/viewHierarchy/renderingSystem.tsx#L19